src/components/NetworkSelector/styles.scss

```diff
@@ -125,6 +126,23 @@
         border-radius: pxToRem(6px);
         border: 1px solid var(--sds-clr-gray-06);
         padding: pxToRem(12px);
+
+        // TODO: temp fix until SDS is fixed (tooltip overlap)
+        // Adjusting form element z-index to fix "Copied" tooltip overlap
+        & > * {
+          &:nth-child(1) {
+            z-index: 4;
+          }
+          &:nth-child(2) {
+            z-index: 3;
+          }
+          &:nth-child(3) {
+            z-index: 2;
+          }
+          &:nth-child(4) {
+            z-index: 1;
+          }
+        }
```

Comment:
This is a hacky temporary fix for the copy tooltip overlap in input if it is followed by another input with a copy button. A proper fix will need to be applied in the SDS, but it might not be a simple one ([issue](https://github.com/stellar/stellar-design-system/issues/254)). Added this to save time right now.

Before

[lab-input-copy-tooltip-overlap-before.webm](https://github.com/user-attachments/assets/a6bd995e-aff8-4c62-9a78-6e53565028e0)

After

[lab-input-copy-tooltip-overlap-after.webm](https://github.com/user-attachments/assets/a69eb1cb-40c6-4bf4-93d6-ed217a860cb7)